src/App.tsx

```diff
@@ -35,7 +35,7 @@ class App extends React.Component <{}, State> {
       .then(response => {
         this.setState({
           goods: response.data
-            .sort((a: Elements, b: Elements) => a.name.localeCompare(b.name))
+            .sort((a: Good, b: Good) => a.name.localeCompare(b.name))
```

Comment:
```suggestion
            .sort((a, b) => a.name.localeCompare(b.name))
```
Якщо опишеш що повертає функція, то можна не вказувати тип кожного з параметрів [ось приклад](https://mateacademy.slack.com/archives/G011BRK1WBH/p1594281273284800)

Reply:
Ось тому, я й плачу